riff-raff/app/controllers/Application.scala

```diff
@@ -78,7 +78,7 @@ class Menu(config: Config) {
         SingleMenuItem(
           "Authorisation",
           routes.Login.authList,
-          enabled = config.auth.allowList.useDatabase
+          enabled = false
```

Comment:
This change hides this menu item:

<img width="1000" height="220" alt="image" src="https://github.com/user-attachments/assets/73404645-d8ad-4fe3-ae25-de0f018deb2f" />

We'll remove this functionality completely in a follow-up PR ([Trello card](https://trello.com/c/zjX5LW6w/378-remove-admin-pages-and-deprecated-logic-for-soon-to-be-deprecated-riffraff-authorisation-solution)).